docker-berp/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# Builds a docker image used for building most projects in this repo. It's
+# used both by contributors and CI.
+#
+FROM ubuntu:22.04
```

Comment:
Would it not be better to depend on some dotnet variant that is supported / good?

Reply:
Yes. I think that makes sense. I'll check what I can do about this.

Reply:
@luke-hill , I've now taken the dotnet/sdk base image instead of installing dotNet itself. I've tried using just the runtime, but that turned out not to be "enough" for our purpose (doesn't include some tool we need).